ocaml/xapi/xapi_pool.ml

```diff
@@ -1857,12 +1857,7 @@ let eject_self ~__context ~host =
     write_first_boot_management_interface_configuration_file () ;
     Net.reset_state () ;
     Xapi_inventory.update Xapi_inventory._current_interfaces "" ;
-    debug "Pool.eject: deleting Host record (the point of no return)" ;
-    Create_misc.create_pool_cpuinfo ~__context ;
-    (* Update pool features, in case this host had a different license to the
-       		 * rest of the pool. *)
-    Pool_features.update_pool_features ~__context ;
-    (* and destroy my control domains, since you can't do this from the API [operation not allowed] *)
```

Comment:
we presumably want both the ejected host and the remaining host to recalculate their pool features? I assume that the ejectee will do this when it reboots

Reply:
Indeed, the ejectee will do this when it reboots. But this is for its own (new and separate) pool  as it has been separated from the others.